src/api/eventTypes.js

```diff
@@ -449,7 +454,13 @@ export type RealmUserUpdateEvent = {|
         // the collections of users and bots in the initial data. Ignore.
         // avatar_source: string,
         // avatar_url_medium: string,
-      |},
+      |}
+    | {|
+        // Digested from `is_active: true`. We add the full user details
+        // from our data structures; the event only carries the user ID.
+        +userToActivate: User,
+      |}
+    | {| +userToDeactivate: User |}, // from `is_active: false`
```

Comment:
Sure, `User` is fine. I typed `UserOrBot` out of habit because that's what we almost always want; but here since we're actually updating these data structures that have a difference, `User` is accurate.